client/src/lib/dependency-injection/graphql/buildReadQuery.js

```diff
@@ -1,9 +1,9 @@
 import { defaultTag } from './tags';
-import { getPluralName, getVariables, getRootParams, getFields, getFragments } from './helpers';
+import { getPluralName, getVariables, getParams, getFields, getFragments } from './helpers';
 
 const buildReadQuery = (tag = defaultTag) => (
   tag`query Read${getPluralName}${getVariables} {
-    read${getPluralName}${getRootParams} {
+    read${getPluralName}${getParams} {
       ${getFields}
```

Comment:
root params is nonsensical - and would result in parameters being set and used in `getVariables` but then never being used as actual parameters - so the variable values wouldn't be used.